config/packages/security.yaml

```diff
@@ -0,0 +1,60 @@
+security:
+  enable_authenticator_manager: true
+  # https://symfony.com/doc/current/security.html#where-do-users-come-from-user-providers
+  providers:
+    accounts:
+      entity:
+        class: 'CsrDelft\entity\security\Account'
+
+  firewalls:
+    dev:
+      pattern: ^/(_(profiler|wdt)|css|images|js)/
+      security: false
+    # Achter deze firewall zitten requests voor de api.
+    api:
+      pattern: ^/API/2.0/
+      stateless: true
+      custom_authenticators: [CsrDelft\service\security\ApiAuthenticator]
+      provider: accounts
+      lazy: true
+    # Achter deze firewall zitten requests die met een private token worden opgehaald.
+    # Concreet zijn dit de forum rss en de agenda ics. Als een request een {private_auth_token}
+    # veld bevat wordt deze opgepikt door deze firewall. Zie ook PrivateTokenAuthenticator
+    private_token:
+      request_matcher: CsrDelft\service\security\PrivateTokenAuthenticator
+      provider: accounts
+      stateless: true
+      custom_authenticators: [CsrDelft\service\security\PrivateTokenAuthenticator]
+    # Dit is de standaard firewall en wordt gebruikt als geen van de andere firewalls gebruikt worden.
+    main:
+      form_login:
+        enable_csrf: true
+        check_path: app_login_check
+      custom_authenticators: [CsrDelft\service\security\WachtwoordResetAuthenticator]
+      lazy: true
+      provider: accounts
+      entry_point: form_login
+      logout:
+        path: app_logout
+        # where to redirect after logout
+        target: default
+      remember_me:
+        secret:   '%kernel.secret%'
+        lifetime: 604800 # 1 week in seconds
```

Comment:
Is een week hoe het voorheen ook was? Ik kan me voorstellen dat een niet-harde-kern-lid minder dan eens per week de stek bekijkt.

Reply:
Het was 30 dagen, maar dat vind ik ook wel een beetje aan de lange kant eigenlijk.

Reply:
Idee voor de toekomst zou zijn dat we daar nog eens een tweedeling in gaan maken: gegevens die je mag zien als 'onveilig lang ingelogd lid' en gegevens waarvoor je je wachtwoord in moet vullen

Reply:
Hoe werkt dit als iemand binnen die week terugkomt? Dan wordt deze tijd niet verlengd, toch?

Reply:
Iedere keer als je de remember token gebruikt om een nieuwe sessie te krijgen wordt ie verlengd. Dus als je eens per week langskomt kun je oneindig ingelogd blijven.